effectiveOfficeBackend/src/main/kotlin/office/effective/features/auth/service/TokenAuthorizer.kt

```diff
@@ -80,4 +87,16 @@ class TokenAuthorizer(private val extractor: TokenExtractor = TokenExtractor())
         logger.trace("Token auth failed with token: {}", tokenString)
         return false
     }
+
+    private fun checkUserAndUpdateAvatar(email: String, newAvatar: String?): Boolean {
+        val result = userFacade.existsByEmail(email)
+        runBlocking {
```

Comment:
[**It is designed to bridge regular blocking code to libraries that are written in suspending style, to be used in main functions and in tests.**](https://kotlinlang.org/api/kotlinx.coroutines/kotlinx-coroutines-core/kotlinx.coroutines/run-blocking.html)